src/py/flwr/common/__init__.py

```diff
@@ -104,6 +105,7 @@
     "Properties",
     "ReconnectIns",
     "RecordSet",
+    "set_logger_propagation",
```

Comment:
Are we expecting users to use this function? If not, let's remove it here and to prevent it from becoming part of our public API

Reply:
Probably note. I'll remove it.